### PR TITLE
[naga] Update naga with new struct members separator

### PR DESCRIPTION
**Pull request by Gordon-F**
**Description**
Update naga version with new struct members separator syntax (https://github.com/gfx-rs/naga/pull/1773)

**Testing**
All updated examples tested on Windows with DX12 backend.

### CODE REVIEW

**Comment by Gordon-F** [comment]
CI error looks unrelated with PR changes:
```
--- STDERR:                           wgpu::example/shadow shadow ---
error: XDG_RUNTIME_DIR not set in the environment.
[2022-03-13T11:27:04Z ERROR wgpu_hal::gles::egl] EGL 'eglInitialize' code 0x3001: DRI2: failed to load driver
thread 'shadow' panicked at 'UNEXPECTED TEST PASS: BACKEND', wgpu/examples/shadow/../../tests/common/mod.rs:294:9
stack backtrace:
   0: rust_begin_unwind
             at /rustc/53cb7b09b00cbea8754ffb78e7e3cb[521](https://github.com/gfx-rs/wgpu/runs/5527286574?check_suite_focus=true#step:13:521)cb8af4b/library/std/src/panicking.rs:493:5
   1: std::panicking::begin_panic_fmt
             at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/std/src/panicking.rs:435:5
   2: shadow::framework::test_common::initialize_test
             at ./examples/shadow/../../tests/common/mod.rs:294:9
   3: shadow::framework::test
             at ./examples/shadow/../framework.rs:458:5
   4: shadow::shadow
             at ./examples/shadow/main.rs:855:5
   5: shadow::shadow::{{closure}}
             at ./examples/shadow/main.rs:854:1
   6: core::ops::function::FnOnce::call_once
             at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/core/src/ops/function.rs:227:5
   7: core::ops::function::FnOnce::call_once
             at /rustc/53cb7b09b00cbea8754ffb78e7e3cb521cb8af4b/library/core/src/ops/function.rs:227:5
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

File: wgpu-hal/examples/halmark/shader.wgsl [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+    size: vec2<f32>,
 };
 
 struct Locals {
-    position: vec2<f32>;
-    velocity: vec2<f32>;
-    color: u32;
+    position: vec2<f32>,
+    velocity: vec2<f32>,
+    color: u32
```
Comment (kvark) [inline wgpu-hal/examples/halmark/shader.wgsl:9]: why omitting the trailing comma here?

File: wgpu/examples/cube/shader.wgsl [hunk 2]
```diff
@@ -1,10 +1,10 @@
 struct VertexOutput {
-    @location(0) tex_coord: vec2<f32>;
-    @builtin(position) position: vec4<f32>;
+    @location(0) tex_coord: vec2<f32>,
+    @builtin(position) position: vec4<f32>
```
Comment (kvark) [inline wgpu/examples/cube/shader.wgsl:3]: same here and everywhere - please keep the comma

**Review by kvark — CHANGES_REQUESTED** [review]

File: wgpu/examples/cube/shader.wgsl (hunk 2 above)
Comment (Gordon-F) [reply wgpu/examples/cube/shader.wgsl]: I returned back all trailing commas, except structs with one member. Just to show that trailing commas are optional.

**Review by kvark — APPROVED** [review]
thank you!

**Pull request merged**